src/drivers/Finnhub.java

```diff
@@ -0,0 +1,267 @@
+package drivers;
+
+import entities.CompanyInformation;
+import entities.CompanyNews;
+import entities.PricePoint;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import use_cases.APIAccessInterface;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.time.*;
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO
+// code the code using okhttp to necessary information about the stocks
+public class Finnhub implements APIAccessInterface {
+    String APIKEY;  // Question: should this be private or static or final or some combination of these things?
+    int NUM_ARTICLES = 5;
+    String RESOLUTION = "D";
+
+    public Finnhub() {
+        File f = new File("file.txt");
+        if(!f.exists()) {
+            // do something
+            System.out.println("please create a file in the src parent directory called file.txt with your Finnhub token");
+            // Question: should we be throwing FileNotFoundException here?
```

Comment:
I think that is unnecessary as the output is clear enough, and the setup instructions take care of this potential issue.

Reply:
Yeah, because the catch FileNotFoundException technically "handles" this case anyway, right?